crates/frontend/src/components/override_form.rs

```diff
@@ -18,30 +20,25 @@ pub fn override_form<NF>(
     #[prop(into, default = None)] handle_key_remove: Option<Callback<String, ()>>,
 ) -> impl IntoView
 where
-    NF: Fn(Map<String, Value>) + 'static,
+    NF: Fn(Vec<(String, Value)>) + 'static,
 {
     let default_config = StoredValue::new(default_config);
-    let (overrides, set_overrides) = create_signal(overrides.clone());
-    let unused_config_keys = Signal::derive(move || {
-        default_config
-            .get_value()
-            .into_iter()
-            .filter(|config| !overrides.get().contains_key(&config.key))
-            .collect::<Vec<DefaultConfig>>()
+    let (overrides, set_overrides) = create_signal(overrides);
```

Comment:
```suggestion
    let (overrides_rs, overrides_ws) = create_signal(overrides);
```

Reply:
https://docs.rs/leptos/latest/leptos/struct.Signal.html#examples

Leptos also uses the same naming convention, i.e. (override for read signal and set_overrides for write signal)

Reply:
I prefer this, so that its not `set_overrides.set(`. This kind of function call looks weird. It is probably a nitpick, I think it makes code easier to read too since I know clearly that this is a signal and not a resource or anything else